zipkin-cassandra/src/main/scala/com/twitter/cassie/connection/ClusterClientProvider.scala

```diff
@@ -141,7 +141,11 @@ private[cassie] class ClusterClientProvider(
         // call login
         val loginMap: java.util.Map[String, String] = HashMap("username" -> username, "password" -> password)
         val authRequest = new AuthenticationRequest(loginMap)
-        client.login(authRequest)
+        val future = client.login(authRequest)
+        future onFailure { exception =>
+          throw exception
+        }
```

Comment:
unsure how to test this block, as the exceptions get buried inside of futures inside of futures.  I think [ScalaFutures](http://doc.scalatest.org/2.0/index.html#org.scalatest.concurrent.Futures) allow for testing of 1 level of futures but its not enough.  Not to mention these are twitter futures, not scala.  Found a [posting](http://cjwebb.github.io/blog/2015/02/02/scalatest-futures/) about how to convert the futures but I didn't get this to work.
I manually verified that the Auth exceptions which we're trying to catch are shown in the output.